decidim-elections/app/views/decidim/elections/admin/answers/index.html.erb

```diff
@@ -30,6 +30,14 @@
                 <%= answer.linked_resources(:proposals, "related_proposals").count %>
               </td>
               <td class="table-list__actions">
+                <% if question.total_votes.positive? %>
+                  <% if answer.selected %>
+                    <%= icon_link_to "circle-check", unselect_election_question_answer_path(election, question, answer), t("answers.select.disable", scope: "decidim.elections.admin"), class: "action-icon--success" %>
+                  <% else %>
+                    <%= icon_link_to "circle-check", select_election_question_answer_path(election, question, answer), t("answers.select.enable", scope: "decidim.elections.admin"), class: "action-icon" %>
```

Comment:
I may lack of context but these are supposed to use different icons, right?

Reply:
We use different colours for it. But from an accessibility aspect, different icons might make sense. I'll add a different icon.